pdu/UDH.go

```diff
@@ -15,27 +17,35 @@ import (
 type UDH []InfoElement
 
 // UDHL return the length (number of octet) of the encoded UDH itself
-func (u UDH) UDHL() (l int) {
-	for i := range u {
-		l += len(u[i].Data)
+func (u *UDH) UDHL() int {
+	if u == nil {
```

Comment:
Don't need this check

Reply:
```go
var *UDH = nil

*UDH.UDHL() // the expect panic ?
```

Reply:
@septs it's invoker/user responsibility for handling nil UDH, not us. Struct function has nothing todo with self checking nil.